_articles/assessments.md

```diff
@@ -0,0 +1,89 @@
+
+Bridge has a shared assessment library of validated, high-quality assessments that researchers can include in their apps. Once copied for use in an app, they can be scheduled as part of a study design. The assessment also indicates specific fields that can be customized for your app, without undermining the scientific validity of the assessment. 
+
+## Using assessments
+
+Curated assessments are available through our [Shared Assessments APIs](/swagger-ui/index.html#/Shared%20Assessments). In addition to some basic information about the assessment, there will be a set of [ExternalResource](/model-browser.html#ExternalResource) objects that provide links to many kinds of documentation. Those links will be assigned to one of the following categories:
+
+|Category|Purpose of Documentation|
+|--------|-----------|
+|customization_options|Describes the fields that can be edited, through a special API for this purpose, to customize an assessment in a specific app. These fields are chosen by the assessment authors because they are not believed to effect the validity of the data collected by the assessment.|
+|data_repository|Describes an online location where data can be found that has been generated by this assessment, in one or more studies.|
+|science_documentation|Documentation of the scientific interpretation of the data collected by the assessment. Schema information, like data dictionaries, falls into this category.|
+|developer_documentation|Documentation on the software and the assessment configuration that is specifically for implementing an assessment in app development (information about the interpretation of the data that is generated should be marked as "science_documentation."|
+|license|The software or other licenses for this assessment.|
+|publication|A publication that was based at least in part on data collected through this assessment.|
+|release_note|Documentation of changes in a specific revision of an assessment (we suggest that the `minRevision` and `maxRevision` values on this `ExternalResource` should both be set to the revision being documented).|
+|sample_app|A sample app showing the implementation of the assessment in working software. Intended for developers who are implementing a new app.|
+|sample_data|Sample data for researchers  that is identical to the data generated by the assessment.|
+|screenshot|An image file (`format` field should be an image MIME type) showing an example of the assessment’s UI within Bridge’s app SDKs.|
+|see_also|Link to a related assessment, measure, or methodology (it does not need to be in the Bridge system).|
+|used\_in\_study|A link to a study that has utilized this assessment (this is a link to some public face for the study, not data or publications from that study).|
+|website|Any other website.|
```

Comment:
Might be useful to suggest that if someone wants to include a video preview of their task, that they could upload to youtube as a public video and include in this field

Reply:
Should we include a video_preview category?